flocker/ca/_script.py

```diff
@@ -90,6 +90,62 @@ def getSynopsis(self):
 
 
 @flocker_standard_options
+class UserCertificateOptions(PrettyOptions):
+    """
+    Command line options for ``flocker-ca create-api-certificate``.
+    """
+
+    helptext = """Create a new certificate for an API end user.
+
+    Creates a certificate signed by a previously generated certificate
+    authority (see flocker-ca initialize command for more information).
+
+    Required parameters:
+
+    * name: A username for which the certificate should be created.
+    """
+
+    synopsis = "<name> [options]"
+
+    optParameters = [
+        ['inputpath', 'i', os.getcwd(),
+         'Path to directory containing root certificate.'],
+        ['outputpath', 'o', os.getcwd(),
```

Comment:
:question: 

I think Jon had a concern about variable defaults which sounded kind of reasonable to me.  You could make the default `None` and write words in the description saying the default is the current directory.